Brewfile

```diff
@@ -1,2 +1,4 @@
+tap 'pact-foundation/pact-ruby-standalone'
 brew 'carthage'
 brew 'swiftlint'
+brew 'pact-foundation/pact-ruby-standalone/pact-ruby-standalone'
```

Comment:
This _should_ be a simple `brew pact-ruby-standalone` since it's already tapped 3 lines above?

Reply:
Yeah, true. I was getting a conflict locally because I'd installed the surpher version as well. I'll change it.